### ISSUE TITLE
EU organizations not supported

### ISSUE THREAD

**Issue by Sobuno** [Aldaviva/PagerDuty#2]
My application is failing to create an alert. When looking into the code, I see the URL `https://events.pagerduty.com/v2/enqueue` is hardcoded. For EU-based applications, this URL should be `https://events.eu.pagerduty.com/v2/enqueue` (note the additional `.eu` in the URL)

**Comment by Aldaviva** [comment]
Thanks @Sobuno.

**Comment by Aldaviva** [comment]
Resolved in 1.1.0.
- [Package on NuGet Gallery](https://www.nuget.org/packages/PagerDuty/1.1.0)
- [Feature documentation](https://github.com/Aldaviva/PagerDuty#base-url)
- Usage example:
    ```cs
    using IPagerDuty pagerDuty = new PagerDuty("my integration key") {
        BaseUrl = new Uri("https://events.eu.pagerduty.com/v2/")
    };
    ```
